pkg/views/initial/view.go

```diff
@@ -171,5 +170,6 @@ func GetCommand() (string, error) {
 		return m.choice, nil
 	}
 
-	return "", errors.New("no command selected")
+	// return on ctrl+c
+	return "", nil
```

Comment:
Shouldn't this also return `common.ErrCtrlCAbort` and then you check it in `RunInitialScreenFlow` @ `pkg/cmd/cmd.go`? If yes, remove the comment as well